src/Microsoft.AspNetCore.Dispatcher.Abstractions/IDispatcherFeature.cs

```diff
@@ -10,5 +10,7 @@ public interface IDispatcherFeature
         Endpoint Endpoint { get; set; }
 
         RequestDelegate RequestDelegate { get; set; }
+
+        DispatcherValueCollection Values { get; set; }
```

Comment:
This looks suspect

Reply:
Let's discuss this. We need to make a call about whether or not route values are first class